README.md

```diff
@@ -30,14 +30,50 @@ The MotorEase code and data has been permanently archived on Zenodo at: (https:/
 
 - This project uses a GloVe embedding for textual similarities. However, GloVe embedding files are large and difficult to host on GitHub. Therefore, we have created a sampleGlove.txt file within the docker container to act as a dummy GloVe model in place of a real one. This text file is formatted the exact same way as a Glove model is normally formatted.
 
-<ins>Running the Image: </ins>
+<ins> Build 1: Running the Image with Local Build: </ins>
 
-- In order to run the image on the container, run this command ``` docker run -it --rm -v $(pwd)/container_files:/container_files itsarunkv/motorease-arm /bin/bash ``` (be sure to use the correct "and" or "arm" image name in this command).
+- With these instructions you will be able to run the Docker Image locally and propagate your changes to the Docker Container automatically.
+
+- This build uses the: glove.6B.100d from ```https://github.com/stanfordnlp/GloVe?tab=readme-ov-file#download-pre-trained-word-vectors```so please download the glove embedding here if you would like to use the same model.
+  - Other models can be used and to use those models please change MotorEase.py in the Code directory with the model you would like to use.
```

Comment:
Would you be able to specify the line numbers like it does in Build 3 instructions to make this easier to understand?

Reply:
Ah okay I added to clarify it at the top of Build 2 that the instructions are meant for WSL and added your note on how it works for PowerShell:
![image](https://github.com/hemkan/MotorEase/assets/93550993/29ffc5cc-c345-44db-9efc-a7e4c2ae2c80)

Reply:
Alrighty I added the line number and directory so it's more obvious :)
![image](https://github.com/hemkan/MotorEase/assets/93550993/9be45f5b-bbf1-484b-8e6e-2e6419ed984e)